src/Internal/Testnet/Contract.purs

```diff
@@ -304,6 +305,13 @@ execDistrFundsPlan withCardanoCliUtxos rounds = do
     )
     roundsFixed
 
+newtype KeyWalletShow = KeyWalletShow KeyWallet
```

Comment:
What's the purpose?

Reply:
Where is it used?

Reply:
This is a wrapper for KeyWallet with a Show instance, which is needed for better DistrFundsError messages